src/Gallifrey.UI.Modern/MainViews/ControlButtons.xaml.cs

```diff
@@ -38,6 +38,18 @@ private void AddButton(object sender, RoutedEventArgs e)
             ModelHelpers.OpenFlyout(new AddTimer(ModelHelpers, startDate: startDate));
         }
 
+        private void CopyButton(object sender, RoutedEventArgs e)
+        {
+            var selectedDate = ViewModel.TimerDates.FirstOrDefault(x => x.IsSelected);
+            var selectedTimer = selectedDate?.Timers.FirstOrDefault(y => y.IsSelected);
+            var jiraRef = selectedTimer?.Reference;
+
+            if (jiraRef != null)
```

Comment:
If you do:

ModelHelpers.GetSelectedTimerIds()
will give you all the selected as an IEnumerable.

Save having this finding the selected logic twice :)